### ISSUE TITLE
Release numbers for the benchmarks againts other frameworks

### ISSUE THREAD

**Issue by Botev** [autumnai/collenchyma#44]
Hi,
I watched the talk on Rust Berlin, however I did not find any place where there are actually published numbers about the speed performance of the framework. Also on the talk it was said that it is twice faster than Tensorflow and Caffee, however it was not specified on which device, as well as I did not hear comments for comparison with Torch, Theano and MXNet which are defacto the current lead in performance in ML. It would be really great if there could be something that we can actually see as numbers as well as the code for the other frameworks against which you are benchmarking.

**Comment by hobofan** [comment]
We mostly benchmarked the [popular convnet-benchmarks](https://github.com/soumith/convnet-benchmarks).

We have some numbers that we got by running it on our local machine here: https://github.com/autumnai/leaf/tree/dump-0.2.0#benchmarks (code is under `example/benchmarks.rs`).
The tests were run on the GPU with cuDNNv3, similar to the other frameworks. The machine contains a `Intel i7-2600k and a NVIDIA Titan X`.

We realize the the benchmarks are not perfect yet, and we will create a PR to the `convnet-benchmarks` repo after the release Leaf 0.2, so it can be properly peer-reviewed and representative numbers can be produced.

It would also be great to have other benchmarks, since I think there are some use-cases where I think Leaf can be significantly faster e.g. one-off inference and network creation time.

There is a issue (https://github.com/autumnai/leaf/issues/26) dedicated for tracking cross-framework benchmarks, but I just saw that it is completely empty :sweat_smile:
